### PR TITLE
Revert "Revert "BucketSharing pre konkretnych userov""

### PR DESCRIPTION
**Pull request by fejjo**
Reverts keboola/storage-api-php-client#351
JIRA Issue: https://keboola.atlassian.net/browse/KBC-14
Main PR: https://github.com/keboola/connection/pull/2277

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -900,6 +900,60 @@ This operations is available only for organization administrators.
                  }
             }
 
+### Share Bucket To Users [POST /v2/storage/buckets/{bucket_id}/share-to-users]
+[Enables sharing](https://help.keboola.com/storage/buckets/sharing/) or **update sharing** for this bucket to a specific set of users in the organization.
+Any user who is included in the targetUsers array will be able to link to this bucket.
```
Comment (ErikZigo) [inline apiary.apib]:
```suggestion
All users who are included in the targetUsers array will be able to link to this bucket.
```

File: apiary.apib [hunk 2]
```diff
@@ -900,6 +900,60 @@ This operations is available only for organization administrators.
...
+This operations is available only for organization administrators.
+
++ Parameters
+    + bucket_id (required) - Bucket Id
+
++ Attributes
+    + targetUsers[] (required, array) - array of users id or user emails
```
Comment (ErikZigo) [inline apiary.apib]:
```suggestion
    + targetUsers[] (required, array) - array of users id or user emails. Users must be member of at least one project in the organization.
```

File: apiary.apib [hunk 3]
```diff
@@ -900,6 +900,60 @@ This operations is available only for organization administrators.
...
+                                   "email": "dev@keboola.com"
+                           },
+                           {
+                                   "id": 2,
+                                   "name": "dev2@keboola.com"
+                           },
+                           {
+                                   "id": 3,
+                                   "name": "dev3@keboola.com"
+                           }
```
Comment (ErikZigo) [inline apiary.apib]: Tady chybí jména těch userů. Ty to vrací taky.

Comment (fejjo) [reply apiary.apib]: Upravene

**Pull request merged**